barpyrus/conky.py

```diff
@@ -45,6 +45,63 @@ def update_label(self, line):
     def eventinputs(self):
         return [ self.conky ]
 
+
+class ConkyMatch:
+
+    def __init__(self, varname):
+        assert isinstance(varname, str), repr(varname)
+        self.varname = varname
+        self.value = None
+        self.operator = None
+
+    def __repr__(self):
+        return '<ConkyMatch varname=%r value=%r operator=%r>' % (self.varname, self.value, self.operator)
+
+    def _stringify(self, value):
+        if isinstance(value, str):
+            return '"%s"' % value.replace('\\', r'\\').replace('"', r'\"')
```

Comment:
I ended up using raw strings for the replacement values, to avoid the [leaning toothpick syndrome](https://en.wikipedia.org/wiki/Leaning_toothpick_syndrome) a bit :wink:

Reply:
Sure, thanks for fixing! Initially, I also tried to write raw strings, but it seems I don't understand them yet. (I tried writing `r'\'` which was rejected by python and so just went safe)